src/Sarif.UnitTests/Converters/CppCheckConverterTests.cs

```diff
@@ -52,8 +52,8 @@ public void CppCheckConverter_HandlesEmptyErrorsElement()
         {
             const string source = "<results> <cppcheck version=\"12.34\" /> <errors>   </errors> </results>";
             const string expected = @"{
-  ""$schema"": ""http://json.schemastore.org/sarif-1.0.0-beta.5"",
-  ""version"": ""1.0.0-beta.5"",
+  ""$schema"": ""http://json.schemastore.org/sarif-1.0.0"",
+  ""version"": ""1.0.0"",
```

Comment:
Do something similar to what was done in JsonTests (can you derive from JsonTests?)